pkg/tinyusb/doc.txt

```diff
@@ -0,0 +1,79 @@
+/**
+ * @defgroup pkg_tinyusb TinyUSB package
+ * @ingroup  pkg
+ * @brief    Provides the tinyUSB stack as package
+ * @author   Gunar Schorcht <gunar@schorcht.net>
+ * @see https://github.com/hathach/tinyusb
+ *
+ * # TinyUSB
+ *
+ * tinyUSB is an open-source cross-platform USB Host/Device stack for
+ * embedded systems.
+ *
```

Comment:
How does tinyUSB relate to the usbus native RIOT modules? Is it (or can it be) used on top of the existing usbus? Or is it mutually exclusive? (E.g. because the set of supported hardware is distinct -- and what when one gains support for a device another has?)

Reply:
tinyUSB and USBUS are mutually exclusive.
tinyUSB provides both the USB stack and the lowlevel driver whereas usbus depends on `periph_usbdev`
FWICT, tinyUSB has more features and support more lowlevel peripherals than usbus/periph_usbdev but as @gschorcht pointed out [here](https://github.com/RIOT-OS/RIOT/pull/18592#issuecomment-1260513518), it looks like tinyUSB uses more RAM/ROM than our stack.

Reply:
> How does tinyUSB relate to the usbus native RIOT modules?

As @dylad said, tinyUSB includes the USB host and device stack as well as device drivers for a variety of platforms. Also, the stack supports many more device classes than `usbus`.

>  Is it (or can it be) used on top of the existing usbus? Or is it mutually exclusive? (E.g. because the set of supported hardware is distinct -- and what when one gains support for a device another has?)

tinyUSB device driver use the same hardware as `periph_usbdev`. So either tinyUSB or `usbus` can be used. But, theoretically, it should be possible to implement an interface to use the tinyUSB stack on-top of a `periph_usbdev` device driver or to use `usbus` on top of a tinyUSB device driver. This would require to map the interface of tinyUSB device drivers to the `periph_usbdev` interface.